en-US/Downloads.md

```diff
@@ -85,6 +85,12 @@ lang: en-US
       <div class="section-body">
         <div class="row">
           <div class="col-md-6">
```

Comment:
This can't be the 5th in a row of 4 - if you want it to be first in the other downloads section, please bump the last item in this row to the next row